File: modulus/Modulus1.java
//Silas Bartha, Sept. 4, 2019, Modulus Review 1
package modulus;

public class Modulus1 {
	public static void main(String[] args) {
		
		//Print 2 columns, 0-100 in one, 0-7 repeating in the other.
		for(int i = 0; i<=100; i++) {
			System.out.printf("%-5d %-5d \n", i, i%8);
		}
	}
}

File: bin/Python Clones/modulus_1.py
# Silas Bartha, Sept. 18, 2019, Modulus Review 1 Clone
# Print 2 Columns, 0-100 in one, 0-7 repeating in the other.
for i in range(100):
    print(f'{i:<5d}{i%8:<5d}')
